.github/workflows/rust.yml

```diff
@@ -0,0 +1,50 @@
+name: Rust-Datafusion
```

Comment:
Is there a reason these checks need a separate workflow and can't be handled in the [pre-commit workflow](https://github.com/dask-contrib/dask-sql/blob/main/.github/workflows/style.yml)?

Reply:
Not that I can think of. I'll move them to pre-commit

Reply:
@charlesbluca given my last comment what do you think? Should I move the logic to style.yaml or leave it in rust.yaml?